mparray/special/_special.py

```diff
@@ -31,22 +31,27 @@
 
 
 @vectorize
-def gammainc(x, a):
-    return mp.gammainc(x, a=0, b=a, regularized=True)
+def gammainc(a, x):
+    return mp.gammainc(a, a=0, b=x, regularized=True)
 
 
 @vectorize
-def gammaincc(x, a):
-    return mp.gammainc(x, a=a, b=mp.inf, regularized=True)
+def gammaincc(a, x):
+    return mp.gammainc(a, a=x, b=mp.inf, regularized=True)
 
 
 @vectorize
 def ndtri(x):
+    if x == 0:
+        return -mp.inf
+    if x == 1:
+        return mp.inf
+    if x < 0 or x > 1:
+        return mp.nan
```

Comment:
Good call. An improvement to the infrastructure might be to have a standard way to define the domain, what to return for edge cases, and what to return for out-of-bounds cases. Then we could test those cases.

Reply:
Yeah. That’s a good idea I think.